subprojects/architecture-test/src/changes/accepted-public-api-changes.json

```diff
@@ -16,6 +16,14 @@
                 "Abstract method has been added in implemented interface"
             ]
         },
+        {
+            "type": "org.gradle.api.AntBuilder",
+            "member": "Superclass [removed]",
+            "acceptation": "org.gradle.api.AntBuilder now extends groovy.ant.AntBuilder",
+            "changes": [
+                "Superclass has been removed"
```

Comment:
This string comes from japicmp (or the Gradle plugin) itself, correct?

Reply:
Yes.